docs/build.sbt

```diff
@@ -59,10 +60,24 @@ lazy val docs = project
     markdownStageIncludeWebJars := false,
     markdownSourceUrl := Some(url(s"https://github.com/lagom/lagom/edit/$branch/docs/manual/"))
 
-  ).dependsOn(serviceIntegrationTestsJavadsl, persistenceJdbcJavadsl,
-      serviceIntegrationTestsScaladsl, persistenceCassandraScaladsl, persistenceJdbcScaladsl, testkitJavadsl,
-      testkitScaladsl, brokerScaladsl, playJson, kafkaBroker, pubsubScaladsl, immutables % "test->compile", 
-      theme % "run-markdown", devmodeScaladsl)
+  )
+  .dependsOn(
+    serviceIntegrationTestsJavadsl,
+    persistenceJdbcJavadsl,
+    persistenceJpaJavadsl,
```

Comment:
This line (adding `persistenceJpaJavadsl`) is the change, but I broke it down into one dependency per line to make it easier to read diffs in the future.